Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
+
 import './App.css';
 
+import CalcDisplay from './components/DisplayComponents/CalculatorDisplay';
+import GreyButton from './components/ButtonComponents/NumberButton';
+import RedButton from './components/ButtonComponents/ActionButton';
+
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+
+
+    <div className="calculator-container">
+
+      <div className="display"><CalcDisplay /></div>
+
+      <div className="keypad">
+        <div className="grey">
+          <div>
+            <GreyButton text={"Clear"} type={"grey-number wide-button"}/>
+          </div>
+
+          <div className="numbers">
+          <GreyButton text={"7"} type={"grey-number"}/> <GreyButton text={"8"} type={"grey-number"}/> <GreyButton text={"9"} type={"grey-number"}/>
+          <GreyButton text={"6"} type={"grey-number"}/> <GreyButton text={"5"} type={"grey-number"}/> <GreyButton text={"4"} type={"grey-number"}/>
+          <GreyButton text={"3"} type={"grey-number"}/> <GreyButton text={"2"} type={"grey-number"}/> <GreyButton text={"1"} type={"grey-number"}/>
+          </div>
+          
+          <div>
+            <GreyButton text={"0"} type={"grey-number wide-button"}/></div>
+        </div>
+
+        <div className="red">
+          <RedButton text={"/"} type={"red-action"}/> <RedButton text={"x"} type={"red-action"}/> <RedButton text={"-"} type={"red-action"}/>
```

Comment:
If you write your type attribute like `type={"grey-number red-action"}`, you don't need to specify the `height: 50px;` again in your .red-action class. Specificity win. 👍